### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by Drod86**
<h2>In this branch I:</h2>

- Copied the code for the Microverse trial project volunteered by [AsciencioAlex](https://github.com/AsciencioAlex)
- Added acknowledgement of [AsciencioAlex](https://github.com/AsciencioAlex) in README.md

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```
Comment (Drod86) [inline index.html:4]: Please the viewport meta tag. This is a best practice that helps provide a consistent experience over the largest range of browsers and screen sizes.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    color: #333;
+    margin: 0;
+  }
+  
+  h1 {
+    font-size: 4rem;
+    margin-bottom: 0;
+  }
+  
+  @media (max-width: 460px) {
```
Comment (Drod86) [inline css/style.css:22]: Consider using 'min-width' rather than 'max-width.' This is a best practice for a modern Mobile-First approach to development.

Comment (Drod86) [inline css/style.css:22]: Consider using 'min-width' in place of 'max-width'. This is a best practice for a modern Mobile-First approach to development.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (Drod86) [inline index.html:3]: Your markup looks good. 
I suggest that you add the meta viewport tag. This is a best practice that creates a consistent experience for a wider range of browsers and viewport sizes.

**Review by Drod86 — COMMENTED** [review]
# Required Changes :recycle:
almost-there
## Hey @Drod86    :wave:
## Highlights :1st_place_medal:
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following some requirements
Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly suggest you make your Pull request more descriptive
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.
# Optional
_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._
Cheers, and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
---------------------------------------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(408 kB)

**Review by Cabraham1 — APPROVED** [review]
Hi @Drod86  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**